media/js/base/mozilla-monitor-button.js

```diff
@@ -12,27 +12,28 @@ if (typeof window.Mozilla === 'undefined') {
 
     var MonitorButton = {};
 
-    MonitorButton.init = function(buttonId){
+    MonitorButton.init = function(){
+        var buttons;
 
-        // check for custom buttonId, used in case of multiple buttons.
-        if(!buttonId){
-            buttonId = 'fxa-monitor-submit';
-        }
+        // Collect all monitor buttons
+        buttons = document.getElementsByClassName('js-monitor-button');
 
-        var monitorButton = document.getElementById(buttonId);
+        // Exit if no valid button in DOM
+        if (buttons[0] === undefined) {
+            return;
+        }
 
-        // fetch request
+        // Exit if no fetch support
         var supportsFetch = 'fetch' in window;
-
-        if (!supportsFetch || !monitorButton) {
+        if (!supportsFetch) {
```

Comment:
Minor performance suggestion: move this further up before the check for `buttons.length`, to save redundant DOM queries in non-supporting browsers?